### DISCUSSION TITLE
How to sendNoteToSelfMessage in jsonRpc?

### DISCUSSION THREAD

**Discussion by jrotus** [AsamK/signal-cli#859] [Q&A] [answered]
Hello,

I'm trying to send a message to my self via jsonRpc. 

Using the "send" method:
`{"jsonrpc":"2.0","method":"send","params":{"recipient":[myNumber],"message":"MESSAGE"},"id":42}`
 I receive the desired message
`{"jsonrpc":"2.0","result":{"timestamp":1641841293966},"id":42}`, but I get an error:
`{"jsonrpc":"2.0","method":"receive","params":{"envelope":{"source":null,"sourceNumber":null,"sourceUuid":null,"sourceName":null,"sourceDevice":null,"timestamp":1641841293966},"error":{"message":"org.whispersystems.libsignal.InvalidMessageException: invalid message Message decryption failed","type":"ProtocolInvalidMessageException"}}}
`
This wouldn't bother me, if there weren't periodical notifications of the form:
`{"jsonrpc":"2.0","method":"receive","params":{"envelope":{"source":"number","sourceNumber":"myNumber","sourceUuid":"myUuid","sourceName":"myName","sourceDevice":thisDevice,"timestamp":1641841296193}}`

Is it possible to depress such an error/notifications?

**Comment by lkwg82** [comment]
Be careful with posting private infos, redact them before! (number)

**Comment by AsamK** [comment, marked as answer]
You can send a note to self message with the `noteToSelf` parameter:

`{"jsonrpc":"2.0","method":"send","params":{"noteToSelf":true,"message":"MESSAGE"},"id":42}`
